src/util/bbb.js

```diff
@@ -0,0 +1,35 @@
+const config = require('../../config.json').bbb;
+const crypto = require('crypto');
+const { XMLParser } = require('fast-xml-parser');
+
+if (!config.api_url.endsWith('/')) config.api_url += '/';
+
+const parser = new XMLParser();
+
+/**
+ * @param {'create' | 'join' | 'isMeetingRunning' | 'getMeetings' | 'getMeetingInfo'} endpoint
+ * @param {URLSearchParams | string} params
+ */
+const bbb = async (endpoint, params) => {
+    const search = new URLSearchParams(params);
+    const checksumString = endpoint + search.toString() + config.api_key;
+    const checksum = crypto.createHash('sha1').update(checksumString).digest('hex');
```

Comment:
This is not documented, but according to https://mconf.github.io/api-mate/, you can also choose a stringer hash for the checksum.

Reply:
Lol. Um, I would love to but it doesn't appear to work I'm afraid. At least against the Senfcall backend, I'm getting `unauthorized` when trying sha256 or sha512. :(